### DISCUSSION TITLE
Wales 20 mph speed limit maxspeed:type tags

### DISCUSSION THREAD

**Discussion by NathanARF** [streetcomplete/StreetComplete#5347] [General]
With the new restricted road speed limit in Wales now 20 mph, shouldn't the maxspeed quest not use `maxspeed:type=GB:nsl_restricted` for 20 mph roads? Someone on the talkGB mailing list suggested `maxspeed:type=CY:nsl_restricted` for 20 mph roads with street lighting.

**Comment by riQQ** [comment]
Could you link the thread on talk-GB? I could only find https://lists.openstreetmap.org/pipermail/talk-gb/2023-July/030525.html, but there's no mention of `maxspeed:type=CY:nsl_restricted` as far as I can see.

**Reply by NathanARF** [reply]
It's not on there yet I think. Only goes up to Oct 29th so far.

**Comment by westnordost** [comment]
`CY` is cyprus. `GB-WLS` is the correct ISO code for Wales.

**Comment by riQQ** [comment]
Thread is here:
https://lists.openstreetmap.org/pipermail/talk-gb/2023-October/030852.html
